astro_2.0/features/CreateProposal/components/CustomFunctionCallContent/components/CustomContent/CustomContent.module.scss

```diff
@@ -103,6 +104,38 @@
   }
 }
 
+.methodSelect {
+  flex: 1;
+  max-width: fit-content;
+
+  button {
+    @extend %title3;
+    border: 0;
+    overflow: hidden;
+    padding-left: 0;
+    padding-top: 12px;
+    white-space: nowrap;
+  }
+
+  .row {
+    font-family: var(--font-family-primary);
+    font-size: 14px;
+    width: 100%;
+  }
+}
+
+.methodSelect ul {
+  @extend %body2;
+  min-width: 260px;
+  [data-hidden-value] {
```

Comment:
Is there a way to do the same with more specific selector?

Reply:
I dnot think so, this is a way how this component works. It uses same componsne for list and selected value but we need some details to be hidden only in control